pkg/apis/admissionregistration/v1/zz_generated.conversion.go

```diff
@@ -0,0 +1,517 @@
+// +build !ignore_autogenerated
+
+/*
+Copyright 2020 Authors of Arktos.
```

Comment:
missing kubernetes copyright

Reply:
This is by design for current arktos_copyright.sh. all new added files only has arktos copyrights if we didn't  add new added files to hack/arktos_copyright_copied_k8s_files.